File: Common/Entity/src/main/java/it/ey/dto/FaseDTO.java
package it.ey.dto;


import com.fasterxml.jackson.annotation.JsonIgnore;
import it.ey.entity.campiTecnici.CampiTecnici;
import lombok.*;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class FaseDTO extends CampiTecniciDTO {

    private Long id;
    private Long idSezione22;
    @JsonIgnore
    private Long idPiao;
    private String denominazione;

    private String descrizione;

    private String tempi;

    //NoSql
    private AttoreDTO attore;

    private AttivitaDTO  attivita;


}

File: PIAO_FE/src/app/shared/models/classes/fase-dto.ts
import { CampiTecniciDTO } from './campi-tecnici-dto';
import { AttoreDTO } from './attore-dto';
import { AttivitaDTO } from './attivita-dto';

export class FaseDTO extends CampiTecniciDTO {
  id?: number;
  idSezione22?: number;
  denominazione?: string;
  descrizione?: string;
  tempi?: string;
  attore?: AttoreDTO;
  attivita?: AttivitaDTO;
}
